dmutils/apiclient/data.py

```diff
@@ -244,3 +144,133 @@ def update_user_password(self, user_id, new_password):
             logger.info("Password update failed for user %s: %s",
                         user_id, e.status_code)
             return False
+
+    # Services
+
+    def find_draft_services(
+            self, supplier_id, service_id=None, framework=None):
+
+        url = "/draft-services?supplier_id={}".format(supplier_id)
+
+        if service_id:
+            url = "{}&service_id={}".format(url, service_id)
+
+        if framework:
+            url = "{}&framework={}".format(url, framework)
+
+        return self._get(url)
+
+    def get_draft_service(self, draft_id):
+        return self._get(
+            "/draft-services/{}".format(draft_id)
+        )
+
+    def delete_draft_service(self, draft_id, user):
+        return self._delete(
+            "/draft-services/{}".format(draft_id),
+            data={
+                "update_details": {
+                    "updated_by": user,
+                    "update_reason": "deprecated",
```

Comment:
Is it worth removing the "update_reason" fields everywhere?  The API completely ignores them now.